docs/source/algorithms/LoadEventNexus-v1.rst

```diff
@@ -67,6 +67,21 @@ The pre-counting step takes some time but that is normally compensated
 by the speed-up in avoid re-allocating, so the net result is smaller
 memory footprint and approximately the same loading time.
 
+Event Compression
+#################
+
+When ``CompressTolerance`` is set, the data loaded will be compressed while creating the individual weighted events.
+The compression will accumulate events to create a weighted event with no time and cannot be further filtered afterwards.
+This parameter is interpreted to be linear tolerance when positive, and logorithmic tolerance when negative.
+When ``CompressBinningMode`` is specified, the ``CompressTolerance`` is modified be linear or logarithmic.
+
+This mode does take longer than running ``LoadEventNexus`` without compression,
+but reduces the overall memory used during the algorithm execution and of the resulting workspace.
```

Comment:
"memory used during the algorithm and of the resulting workspace":  seems awkward to me,
how about something like:
 "memory used during algorithm execution, and memory used by the resulting workspaces"